mobify/progressive-web-scaffold
# Update version of node-sass

On occasion, when building the scaffold I've seen error:

```
    @include susy-breakpoint($tablet-breakpoint, $tablet-layout) {
            ^
          Stack level too deep
          in /Users/kerrmiller/work/01-Astro/progressive-web-scaffold/web/app/containers/checkout-shipping/_checkout-shipping.scss (line 55, column 10)
        at options.error (/Users/kerrmiller/work/01-Astro/progressive-web-scaffold/web/node_modules/node-sass/lib/index.js:286:26)
     @ ./app/main.jsx 27:18-46
     @ multi whatwg-fetch ./app/main.jsx
    Child extract-text-webpack-plugin:
```

A quick google shows that the cause may be that `libsass` had a memory leak as per: https://github.com/sass/node-sass/issues/1669 and https://github.com/sass/libsass/pull/2171.

node-sass v 4.2.0 fixes that memory leak, and 4.5.0 is the latest version.

## Changes
- Upgrade node-sass in package.json

## How to test-drive this PR
- Run `npm update node-sass`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- It should all work
- Also build for prod: `npm run prod:build`
